README.md

```diff
@@ -358,6 +358,23 @@ Templates in React are written inside the JavaScript file using the [JSX languag
 </PrimaryButton>
 ```
 
+### VueJs
+
+Component props can be passed in as strings using a literal syntax, e.g `size="big"` or as actual values using `v-bind`, e.g. `v-bind:disabled="true"`.
```

Comment:
What do you think about writing it like this:

Component props can be passed in as: 
- literal  (using strings) e.g `size="big"` 
- dynamic (using `v-bind` with actual values) e.g. `v-bind:disabled="true"`.

Reply:
I think the bullet points work better, happy to make the change.